examples/asr/asr_streaming/speech_to_text_streaming_infer.py

```diff
@@ -183,7 +205,31 @@ def main():
         action="store_true",
         help="Whether to compare the output of the model with the offline mode.",
     )
-    parser.add_argument("--batch_size", type=int, default=32)
+    parser.add_argument(
+        "--batch_size",
+        type=int,
+        default=32,
+        help="The batch size to be used to perform streaming in batch mode with multiple streams",
+    )
+    parser.add_argument(
+        "--chunk_size",
```

Comment:
Same as below

Reply:
Change argument to be "offline_chunk_size"

Reply:
These parameters are not specific to offline models. For models with limited context, these values have some default values but they can get controlled by the user even for streaming models.